### PR TITLE
glib: Fix invalid types let from_variant panic

### PR DESCRIPTION
**Pull request by sophie-h**

### CODE REVIEW

File: glib/src/variant.rs [hunk 1]
```diff
@@ -642,7 +646,22 @@ where
     V: FromVariant,
     H: BuildHasher + Default,
 {
+    /// ```
```
Comment (GuillaumeGomez) [inline glib/src/variant.rs:649]: Maybe turn it into a code comment? Inner functions are not documented after all. :)

**Review by sdroege — APPROVED** [review]

File: glib/src/variant.rs (hunk 1 above)
Comment (sophie-h) [reply glib/src/variant.rs:649]: Will it still be picked up as a test?

Comment (sdroege) [reply glib/src/variant.rs:649]: No, but you could also add a non-doctest here

```rust
#[test]
fn test_that_thing() {
    [...]
}
```

**Comment by sdroege** [comment]
@GuillaumeGomez Should include that in the next bugfix release too

**Comment by GuillaumeGomez** [comment]
Indeed, just waiting for the test to be moved.

**Review by sdroege — APPROVED** [review]

**Comment by sdroege** [comment]
See https://github.com/gtk-rs/gtk-rs-core/pull/171

**Pull request merged**